modules/gdscript/gdscript_editor.cpp

```diff
@@ -56,15 +56,15 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 
 	String _template = String() +
 					   "extends %BASE%\n\n" +
-					   "# class member variables go here, for example:\n" +
+					   "# Class member variables go here, for example:\n" +
 					   "# var a = 2\n" +
 					   "# var b = \"textvar\"\n\n" +
 					   "func _ready():\n" +
 					   "%TS%# Called when the node is added to the scene for the first time.\n" +
 					   "%TS%# Initialization here.\n" +
 					   "%TS%pass\n\n" +
 					   "#func _process(delta):\n" +
-					   "#%TS%# Called every frame. Delta is time since last frame.\n" +
+					   "#%TS%# Called every frame. 'delta' is time since last frame.\n" +
```

Comment:
Well, let's avoid a 4th PR on those templates and fix the English right away, it should be `'delta` is *the* time since *the* last frame.`, or better (IMO) `'delta' is the elapsed time since the previous frame.`

Reply:
I'd prefer `'delta' is the elapsed time since the previous frame.`